### PR TITLE
Add NLCD datasets

### PR DESCRIPTION
**Pull request by anayeaye**
This draft PR is to coordination the promotion of two NLCD collections to production
- `nlcd-annual-conus`
- `nlcd-new-urbanization`

### CODE REVIEW

**Comment by anayeaye** [comment]
I also uploaded the thumbnail from veda-config [PR 432](https://github.com/NASA-IMPACT/veda-config/pull/432) to https://thumbnails.openveda.cloud/nlcd-cover.png because we do not yet have a standard way to add thumbs to S3

**Comment by anayeaye** [comment]
These collections have been published to staging and production and the custom nlcd colormap is now in the production raster-api so this PR is ready to merge

File: ingestion-data/production/collections/nlcd-new-urbanization.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+    "id": "nlcd-new-urbanization-v2",
```
Comment (smohiudd) [inline ingestion-data/production/collections/nlcd-new-urbanization.json:2]: This id and title aren't the same as the one in the dataset-config

Comment (anayeaye) [reply ingestion-data/production/collections/nlcd-new-urbanization.json:2]: Thanks for catching this! I had temporarily created an additional collection while testing new vs old COGs. I'll fix the metadata in this PR

**Review by botanical — COMMENTED** [review]
Screenshot 2024-12-20 at 9 27 03 AM
`nlcd-annual-conus` looks a bit misaligned and the visualization doesn't appear for `nlcd-new-urbanization`
Screenshot 2024-12-20 at 9 28 27 AM

**Comment by anayeaye** [comment]
> Screenshot 2024-12-20 at 9 27 03 AM `nlcd-annual-conus` looks a bit misaligned and the visualization doesn't appear for `nlcd-new-urbanization` Screenshot 2024-12-20 at 9 28 27 AM

@botanical We have this issue for many collections 😢. Here's the issue to address the problem https://github.com/NASA-IMPACT/veda-backend/issues/420

**Review by botanical — APPROVED** [review]

**Pull request merged**